### DISCUSSION TITLE
scsi2sd to zuluscsi configuration migration

### DISCUSSION THREAD

**Discussion by mmaseda12** [ZuluSCSI/ZuluSCSI-firmware#213] [Q&A] [answered]
Can someone please tell me the proper zuluscsi .ini file setting to replicate my scsi2sd configuration show on this picture.

Thanks in advance,
Manuel

![image](https://github.com/ZuluSCSI/ZuluSCSI-firmware/assets/78750392/a93e8608-3275-492c-a04c-a7a70694b423)

**Comment by aperezbios** [comment, marked as answer]
First, you'd need to create or copy an image file of comparable size, named HD6.img. The ZuluSCSI readme at https://github.com/ZuluSCSI/ZuluSCSI-firmware/blob/main/README.md contains some instructions on how to create an empty zero-byte file of arbitrary size.

Then, create a zuluscsi.ini file, with the following contents, and place it on the SD card used with your ZuluSCSI. You can comfirm it's being read by examining the output of zululog.txt
```
[SCSI6]
Vendor=HP
ProductID=C3010
```

That's it.

**Reply by mmaseda12** [reply]
Thanks,

Wasn’t sure if I needed to add any sector info to the .ini file.  Will give it a try this weekend.

Manuel
